kore/src/Kore/Step/Simplification/And.hs

```diff
@@ -39,7 +39,7 @@ import Kore.Internal.OrPattern
     )
 import qualified Kore.Internal.OrPattern as OrPattern
 import Kore.Internal.Pattern as Pattern
-import Kore.Internal.TermLike
+import Kore.Internal.TermLike as TermLike
```

Comment:
Would this be better with separate imports for the qualified and non-qualified stuff?